src/api-requests/get-articles.ts

```diff
@@ -26,7 +70,7 @@ export const CS_getArticleBySlugFromCMS = async (
         client,
         variables
     )) as CS_ArticleResponse[];
-
+    await CS_adjustOriginalPublishDate(client, calculatedSlug, articles);
```

Comment:
Do we have a sense of how this will affect page speed? Seems like fetching both all articles and all publish metadata might add some time.

Reply:
it will add some time - let me check the metrics - 
there are 2 other options 
1. get from preval - which might be stale
2. combine the graphql query - like a join